src/main/java/dialog/jna/ShTypes.java

```diff
@@ -31,15 +31,15 @@
 public interface ShTypes {
 
     @FieldOrder({ "pszName", "pszSpec" })
-    class COMDLG_FILTERSPEC extends Structure {
-        WString pszName;
-        WString pszSpec;
+    public static class COMDLG_FILTERSPEC extends Structure {
```

Comment:
The `public` modifiers on the fields are necessary, but all `interface` elements are `public` and  `static` by default so these are redundant.